tools/mason/MasonTest.chpl

```diff
@@ -99,12 +108,11 @@ private proc runTests(show: bool, run: bool) {
         }
```

Comment:
Just noticed this on line `105` above: I think it'd make sense for the ` if show then writeln(compCommand);` to occur before the compilation, such that the user sees the command before any potential error (and sees the commands in real-time).